docs/apps/components/color.md

```diff
@@ -0,0 +1,190 @@
+        this.setData(
+         {
+         ""color_o1cfjhoq": "#AEE710",}
+        );
+        ```
+
+## Config
+
+| **Name**|**Description**|**Icon**|
+|---------|---------------|--------|
+|**ID**| Color's unique identifier|<img src= "/apps/components/img/input_id.png">|
+|**Name**| [**Optional**] The display name of the color field|<img src= "/apps/components/img/checkbox_name.png">|
+|**Width**| [**Optional**] Width of the field|<img src= "/apps/components/img/input_width.png">|
+|**Label Width**|Width of the label associated with an input field. It determines the horizontal space occupied by the label text|<img src= "/apps/components/img/input_labelwidth1.png">|
+|**Label Wrap**| If the label is longer than the allowed width the text will continue on another line|<img src= "/apps/components/img/input_labelwrap1.png">|
+|**Hide Label**| Hides the label on the form|<img src= "/apps/components/img/input_hidelabel.png">|
+|**Text Prompt**| A description to aid the user when completing the field|<img src= "/apps/components/img/input_textprompt.png">|
+|**Support Transparency Options**|Ability to specify the transparency level of a color|<img src= "/apps/components/img/color_support.png">
+|**Default Value**| Controls the initial color selection in a color picker, while still allowing the user to change it if desired|<img src= "/apps/components/img/color_default.png">
+|**Custom Class**| An HTML class attribute which allows further customisation [See Form Attribute > Style Sheets](https://bani-appsection--connexcs-docs.netlify.app/apps/page-builder/#form-attribute)|<img src= "/apps/components/img/input_customclass.png">|
+|**Attribute Action**|Enable **Data Binding** to connect the data to UI<br>Enable **Hidden** action to hide the field</br>Enable **Disabled** action to make the field unusable|<img src= "/apps/components/img/checkbox_attributeaction.png">|
+
+### Validation
+
+Form validation is the process of checking the data entered into a form to ensure that it's valid and complete.
+
+This helps to prevent users from submitting forms with invalid data, which can cause problems for the application that's processing the form.
+
+Form validation can be performed using a variety of methods, including:
+
+| **Name**| **Description**|
+|---------|----------------|
+| **Required**| Single line of text|
+|**Custom Validation Rules**|(rule, value, callback) => {|
+||**rule**: Verification rule, you can view the verification configuration information through this parameter; rule.fieldcan get the field identifier of the current verification.|
+||**value**: Value of the current field|
+||**callback**: Callback function (must be called) upon completion of validation; **callback('Error message')/ callback(new Error('Error message'))**. These are two ways to return an error message.|
+
+<img src= "/apps/components/img/cascader_validation.png">
+
+!!! Info
+    1. The **callback()function** is also called to verify success in the custom checkup method.
```

Comment:
Isn't **custom checkup method** supposed to be **custom validation method** instead? Not sure

Reply:
done